### PR TITLE
Fixed vulnerabilities in MSBuildV1 task and updated task-lib-version till 3.3.1

### PR DESCRIPTION
**Pull request by SvetlanaMaliugina**
**Task name**: MSBuildV1
**Description**: Fixed vulnerabilities with ```shelljs``` in MSBuildV1 task.  ```azure-pipelines-task-lib``` version was updated for ```Tasks/Common/MSBuildHelpers common package.```
```MSBuildV1```, ```VSBuildV1```, and ```XamarinAndroidV1``` versions were bumped since these tasks depend on this common package.

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** [Governance alerts] MSBuildV1 task (shelljs) [#3070](https://github.com/microsoft/build-task-team/issues/3070)

**Checklist**:
- [x] Task version was bumped 
- [x] Checked that applied changes work as expected

**Risks analysis checklist:**
* [x] There are no risky dependency updates
* [x] Changes have been tested
* [x] Enough test coverage for changes and current test coverage for tasks doesn't look poor
* [x] We understand how tasks are working, how changes affect tasks behavior
* [x] There are no breaking changes
* [x] There are no other concerns
* [x] I have not discovered any new uncovered test/use cases

### CODE REVIEW

**Review by vmapetr — APPROVED** [review]

**Comment by SvetlanaMaliugina** [comment]
Run test canary pipeline for:
MSBuild task - https://abtttestorg.visualstudio.com/MSBuild_task/_build/results?buildId=5383&view=results
logs:
[logs_5383.zip](https://github.com/microsoft/build-task-team/files/9090800/logs_5383.zip)
VSBuild task -  https://abtttestorg.visualstudio.com/VSBuild_task/_build/results?buildId=5384&view=results
logs: 
[logs_5384.zip](https://github.com/microsoft/build-task-team/files/9090819/logs_5384.zip)

File: Tasks/MSBuildV1/package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^10.17.0",
     "azure-pipelines-task-lib": "^3.1.2",
-    "msbuildhelpers": "file:../../_build/Tasks/Common/msbuildhelpers-1.197.0.tgz"
+    "msbuildhelpers": "file:../../_build/Tasks/Common/msbuildhelpers-2.208.0.tgz"
```
Comment (KonstantinTyukalov) [inline Tasks/MSBuildV1/package.json:18]: Can we use `msbuildHelpers-v3` from npm instead? - https://github.com/microsoft/azure-pipelines-tasks/tree/master/common-npm-packages/MSBuildHelpers-v3

Comment (DenisRumyantsev) [reply Tasks/MSBuildV1/package.json:18]: @KonstantinTyukalov, yes, we can, but in [another PR](https://github.com/microsoft/azure-pipelines-tasks/pull/16433).

**Review by DenisRumyantsev — APPROVED** [review]
Tested it end-to-end, `MSBuildV1` , `VSBuildV1` , and `XamarinAndroidV1` tasks work fine with these changes.

**Review by kirill-ivlev — APPROVED** [review]

**Review by KonstantinTyukalov — APPROVED** [review]

**Pull request merged**